metrics/views.py

```diff
@@ -0,0 +1,190 @@
+                "logged_at": the time of log in this format(2021-01-04 15:27:57)
+                "temp": the temperature
+                "uuid": the uuid of sensor
+
+        GET:
+
+            "list" --> http://192.168.88.50:8000/api/metrics/log/?limit=2&offset=10&sensor=1
+
+                To receive a list of recent logs
+                example result -->
+                                {
+                                    "id": 135,
+                                    "sensor": {
+                                        "sensor_model": "ds18b20",
+                                        "id": 1,
+                                        "title": "sensor_1",
+                                        "description": "This sensor used in room number 2",
+                                        "type": 1,
+                                        "unit": 2
+                                    },
+                                    "logged_at": "2021-01-04T15:58:53.645127Z",
+                                    "temp": 24.875,
+                                    "voltage": null,
+                                    "current": null,
+                                    "resistance": null
+                                },
+
+
+            "retrieve" -->  http://localhost:8000/api/metrics/log/171/
+
+                retrieve complete details of a log
+                example result -->
+                                {
+                                    "id": 171,
+                                    "sensor": {
+                                        "id": 1,
+                                        "sensor_model": "ds18b20",
+                                        "title": "sensor_1",
+                                        "description": "This sensor used in room number 2",
+                                        "type": {
+                                            "id": 1,
+                                            "name": "tempurature"
+                                        },
+                                        "unit": {
+                                            "id": 2,
+                                            "name": "celsius"
+                                        }
+                                    },
+                                    "created_at_by_server": "2021-01-04T12:35:23.457838Z",
+                                    "updated_at_by_server": "2021-01-04T12:35:23.457859Z",
+                                    "logged_at": "2021-01-04T15:58:53.645127Z",
+                                    "temp": 24.875,
+                                    "voltage": null,
+                                    "current": null,
+                                    "resistance": null
+                                }
+    """
+
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = ['sensor']
+
+    def get_queryset(self):
+
+        if self.action == 'create':
+            pass
```

Comment:
In the create action this method not be called at all

Reply:
removed it